sql-plugin/src/main/311until330-all/scala/com/nvidia/spark/rapids/shims/GpuTypeShims.scala

```diff
@@ -80,4 +80,17 @@ object GpuTypeShims {
 
   def csvRead(cv: cudf.ColumnVector, dt: DataType): cudf.ColumnVector =
     throw new RuntimeException(s"Not support type $dt.")
+
+  /**
+   * Spark supports interval type from 320; Spark supports to/from Parquet from 330,
```

Comment:
This still seems wrong, so maybe I'm still missing the context.  If interval types are supported in Spark 3.2, shouldn't we also support the interval types when running on Spark 3.2?  If this predicate function is specific to the Parquet context then the name needs to be updated accordingly.  Otherwise it's too generically named and could be called in other contexts, returning the wrong value for those contexts.

Reply:
Updated, added meaningful methods.